crowbar_framework/config/repos-cloud.yml

```diff
@@ -39,26 +39,6 @@ suse-12.0:
         key_md5: "9a62177e1c6852d48453ed3909956d6b"
       url:
       ask_on_error: false
-    suse-enterprise-storage-2-pool:
-      name: "SUSE-Enterprise-Storage-2-Pool"
-      required: "optional"
-      features: ["ceph"]
-      repomd:
-        tag: "obsproduct://build.suse.de/SUSE:SLE-12:Update:Products:SES2/ses/2/POOL/x86_64"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/products/storage/2/x86_64/product"
-      ask_on_error: false
-    suse-enterprise-storage-2-updates:
-      name: "SUSE-Enterprise-Storage-2-Updates"
-      required: "optional"
-      features: ["ceph"]
-      repomd:
-        tag: "obsrepository://build.suse.de/SUSE:Updates:Storage:2:x86_64/update"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
-      url:
-      smt_path: "SUSE/Updates/Storage/2/x86_64/update"
-      ask_on_error: false
```

Comment:
It will be done in another commit https://github.com/crowbar/crowbar-core/commit/ba5fd6523a7ec1fcb8b8bf135f1124ec2412198c